src/backend/distributed/sql/udfs/citus_stat_activity/latest.sql

```diff
@@ -0,0 +1,42 @@
+-- citus_stat_activity combines the pg_stat_activity views from all nodes and adds global_pid, nodeid and is_worker_query columns.
```

Comment:
There is still a possibility that Postgres drop some column from pg_stat_activity, where we have to drop.

In that case, we cannot do `CREATE OR REPLACE ciitus_stat_activity` because of this: https://www.postgresql.org/message-id/flat/BYAPR16MB2789E8E00904B6ED17FE1955BE9B0%40BYAPR16MB2789.namprd16.prod.outlook.com

In that case, we have to do `DROP VIEW citus_stat_activity CASCADE`. Or, we can keep that column until a major Citus release and return NULL.

I think that is OK